mergekit/architecture.py

```diff
@@ -326,6 +326,57 @@ def sliceable(self) -> bool:
     def has_defined_spaces(self) -> bool:
         return False
 
+class DeepseekV2TensorNames(ArchitectureInfo, BaseModel):
+    ARCHITECTURE_NAME: ClassVar[str] = "DeepseekV2ForCausalLM"
+    num_local_experts: int
+
+    def name(self) -> str:
+        return "DeepseekV2"
+    
+    @classmethod
+    def from_config(cls, config: PretrainedConfig):
+        return DeepseekV2TensorNames(num_local_experts=config.n_routed_experts)
+    
+    def pre_weights(sef, config: PretrainedConfig) -> List[WeightInfo]:
+        return DEEPSEEKV2_INFO.pre_weights(config)
+    
+    def post_weights(self, config: PretrainedConfig) -> List[WeightInfo]:
+        return DEEPSEEKV2_INFO.post_weights(config)
+    
+    def num_layers_config_key(self) -> str:
+        return DEEPSEEKV2_INFO.num_layers_config_key()
+    
+    def layer_weights(
+            self, index: int, config: PretrainedConfig
+    ) -> Optional[List[WeightInfo]]:
+        num_experts = self.num_local_experts
```

Comment:
Nit: Any reason for local aliasing? I see it is also present in the Mixtral code, but any reason you left it in?